docs/reference/elasticsearch/mapping-reference/semantic-text.md

```diff
@@ -359,6 +359,28 @@ PUT test-index
 
 1. Ensures that highlighting is applied exclusively to semantic_text fields.
 
+## Updates and partial updates for `semantic_text` fields [semantic-text-updates]
+{applies_to}
+stack: ga 9.0
```

Comment:
is this behavior specific to 9.0 or was it relevant to prior releases? 

if it is only relevant to 9.0, the tag needs a little cleanup to render
````suggestion
```{applies_to}
stack: ga 9.0
```
````

Reply:
I removed the tag, thanks